rubin_sim/maf/run_comparison/archive.py

```diff
@@ -41,7 +41,7 @@
 
 SUMMARY_SOURCE = os.environ.get(
     "RUBIN_SIM_SUMMARY_SOURCE",
-    "https://raw.githubusercontent.com/lsst-pst/survey_strategy/main/fbs_2.0/summary_2022_11_16.csv",
+    "https://raw.githubusercontent.com/lsst-pst/survey_strategy/main/fbs_2.0/summary_2023_01_01.csv",
```

Comment:
Should this be smart enough to grab the most recent version in case it changes? Or just put the most recent copy in `rubin_sim_data`?

Reply:
Most likely, yes that is something we should do instead. I'm ok with leaving it like this for now .. especially as we probably want to figure out a better versioning setup.